solutions/community-deploy/communitybuild.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Please change this to `Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates`